lib/components/art/ArtDetailWidget.dart

```diff
@@ -11,26 +14,89 @@ class ArtDetailWidget extends StatefulWidget {
 }
 
 class _ArtDetailWidgetState extends State<ArtDetailWidget> {
-  @override
   Widget build(BuildContext context) {
     return Container(
       color: Colors.white,
-      child: Center(
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            Text(
-              widget.data.title,
-              style: TextStyle(
-                  fontSize: 36.0, color: Theme.of(context).primaryColor),
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.start,
+        children: <Widget>[
+          Card(
+            shape: RoundedRectangleBorder(
+              borderRadius: BorderRadius.circular(25.0),
+            ),
+            child: Container(
+              width: MediaQuery.of(context).size.width,
+              height: 250,
+              decoration: BoxDecoration(
+                image: DecorationImage(
+                  fit: BoxFit.fill,
+                  image: NetworkImage(widget.data.imgUrl),
+                ),
+              ),
+            ),
+          ),
+          SizedBox(
+            height: 15.0,
+          ),
+          ListTile(
+            leading: CircleAvatar(
+              backgroundColor: Colors.pink,
+              radius: 25.0,
+            ),
+            title: Text(
+              'John Appleseed',
+              style: GoogleFonts.quicksand(
+                color: Colors.black,
+                fontWeight: FontWeight.w700,
+                fontSize: 16.0,
+              ),
             ),
-            Text(
+            trailing: IconButton(
+              icon: Icon(FontAwesomeIcons.solidBookmark),
```

Comment:
please use the bookmark icon from https://api.flutter.dev/flutter/material/Icons-class.html